tests/searchTest.js

```diff
@@ -31,6 +31,28 @@ describe(testSuite.name, function () {
 
 function _validateResults(testCase, done) {
   var priorityThresh = testCase.priorityThresh || testSuite.priorityThresh;
+  var expectedLocation = testCase.out;
```

Comment:
Functionality wise looks good. I'd pull the expected location lookup into a separate function at this point and have that return a location of an Error. Then here if it's instanceof Error call done with it, if location call request.